### PR TITLE
Add Contribution Guide

### PR DESCRIPTION
**Pull request by lkiesow**
This patch adds a development and contribution guide, explaining how to
best provide contributions for this project and how contributions will
be handles.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+- Bug reports and patch reviews
+
+
+Being Pragmatic
+---------------
+
+For every rule, there is an exception.
+If you find that there is a good reason one of the following rules does not apply to you,
+please bring it up and explain why.
+We like to be pragmatic if necessary instead of blindly follow rules.
```
Comment (benszau) [inline CONTRIBUTING.rst]: I like that 👍

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+
+Merging
+-------
+
+We prefer to `rebase and merge pull requests`_ if the contribution has a sane commit history
+or to `squash and merge your pull request commits`_ if they have not.
+This is meant to keep the overall commit history as clean as possible.
+If you prefer a specific merge mode for any reason, please indicate that on the pull request.
+
```
Comment (benszau) [inline CONTRIBUTING.rst]: For inexperienced contributors, I would propose to add a link to [this contribution explanation](https://github.com/firstcontributions/first-contributions). There you can finde a step-by-step  guide of how to create a fork and how to create a pull request.
```suggestion
If you need more information about how to create a fork or a pull request, we suggest to have a look [here](https://github.com/firstcontributions/first-contributions).

```

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
+Contributing to emissions-api
+=============================
+
+As an open source project, emissions-api welcomes contributions of many forms.
+Examples of contributions include:
+
+- Code patches
+- Documentation improvements
+- Bug reports and patch reviews
```
Comment (benszau) [inline CONTRIBUTING.rst]: I like this introduction text as it invtes the reader get a part of this project.
I wuld like to propose two minor changes:
- Change many to any  `emissions-api welcomes contributions of any form`
- In the example of contributions, one important form is missing -> help developing new features (or is this meant by 'Code patches')

File: CONTRIBUTING.rst (hunk 2 above)
Comment (lkiesow) [reply CONTRIBUTING.rst]: Added this to the top section

**Pull request merged**